app/profile/top/genres/loading.tsx

```diff
@@ -0,0 +1,5 @@
+import { TopGenresSkeleton } from '@app/profile/sections/top-genres.skeleton'
+
+export function TopGenresLoading() {
```

Comment:
Did you even take a look at the docs?

<img width="1384" alt="Screenshot 2023-12-26 at 12 21 53" src="https://github.com/Rigtch/rigtch-fm-www/assets/56691628/ee81c3e3-3825-45ea-90e3-60689bb20b1d">

```suggestion
export default function TopGenresLoading() {
```